ui/doc/2018/db-placement.md

```diff
@@ -0,0 +1,20 @@
+# Shared Header (a.k.a. "Chrome")
+-David Mohs
+
+The following UI design represents Job Manager within the context of Saturn and the Data Biosphere:
```

Comment:
Is this a problem unique to Job Manager? Do we anticipate having other tabs here that are sourced elsewhere? Implementing this twice is a tough sell, but 3 times would be a non-starter, I think.

Reply:
We should establish a common understanding of what is being proposed here. This doc states that we will be implementing the header in the mock you linked above. A fully functional implementation of the above is unlikely to be 130 (80char wrapped) lines of code.

Reply:
Agreed, but based on this size, I'm making the judgement that even a fully-functional implementation would likely not be an unreasonable burden to implement.